spec/vast_trackers.spec.js

```diff
@@ -223,5 +223,36 @@ describe('VASTTracker', function() {
         expect(spyEmitter).toHaveBeenCalledTimes(1);
       });
     });
+
+    describe('setProgress', () => {
+      let spyTrack;
+      beforeEach(() => {
+        spyTrack = jest.spyOn(vastTracker, 'track');
+        vastTracker.assetDuration = 10;
+        vastTracker.setProgress(5);
+      });
+      it('should be defined', () => {
+        expect(vastTracker.setProgress).toBeDefined();
```

Comment:
I'm not sure this test is actually usefull since you call the function setProgress in the beforeEach, it would have thrown an error if it was not defined